index.bs

```diff
@@ -7106,9 +7175,11 @@ reason.
 <div algorithm="create a TransformStream">
  To <dfn export for="TransformStream" lt="set up|setting up">set up</dfn> a
  newly-[=new|created-via-Web IDL=] {{TransformStream}} |stream| given an algorithm <dfn export
- for="TransformStream/set up"><var>transformAlgorithm</var></dfn> and an optional algorithm <dfn
- export for="TransformStream/set up"><var>flushAlgorithm</var></dfn>, perform the following steps.
- |transformAlgorithm| and, if given, |flushAlgorithm|, may return a promise.
+ for="TransformStream/set up"><var>transformAlgorithm</var></dfn>, an optional algorithm <dfn
+ export for="TransformStream/set up"><var>flushAlgorithm</var></dfn>, and an optional algorithm <dfn
+ export for="TransformStream/set up"><var>cancelAlgorithm</var></dfn>, perform the following steps.
+ |transformAlgorithm|, if given, |flushAlgorithm|, and, if given, |cancelAlgorithm|, may return a
```

Comment:
Can we say "if given, |flushAlgorithm| and |cancelAlgorithm|"?

Reply:
Well it's each of them individually are given. Ie `Option<FlushAlgorithm>` and `Option<CancelAlgorithm>` not `Option<(FlushAlgorithm, CancelAlgorithm)>`.

I'm not sure if they can just be coalesced like this. @domenic WDYT?

Reply:
I think @saschanaz's phrasing is correct English and implies `Option<FlushAlgorithm>` and `Option<CancelAlgorithm>`. Consider:

> Dogs, and if present, cats and birds, may bite you in the park.